.teamcity/Pixi/PixiProject.kt

```diff
@@ -48,7 +47,11 @@ object UpdateDependencies : BuildType({
                     {
                       git commit -m "Update pixi.lock"
                       git push -u origin pixi_update_%build.counter%
-                      pixi run --environment default gh pr create --title "[TEAMCITY] Update project dependencies" --body "Teamcity automatically updated the dependencies defined the pixi.toml file. Please verify that all tests succeed before merging" --reviewer JoerivanEngelen,luitjansl
+
+                      echo Teamcity automatically updated the dependencies defined the pixi.toml file. Please verify that all tests succeed before merging. > body.md
+                      echo. >> body.md
+                      type diff.md >> body.md
+                      pixi run --environment default gh pr create --title "[TEAMCITY] Update project dependencies" --body-file body.md --reviewer JoerivanEngelen,luitjansl
```

Comment:
luit jan is not working on this project anymore, so his name can be removed?

Reply:
Changed his name with mine